main_f4.c

```diff
@@ -446,6 +466,53 @@ board_deinit(void)
 	RCC_AHB1ENR = 0x00100000; // XXX Magic reset number from STM32F4x reference manual
 }
 
+#ifdef USE_INTERNAL_OSCILLATOR
+void rcc_clock_setup_hsi_3v3(const struct rcc_clock_scale *clock)
```

Comment:
The libopencm3 doesn't have the a function to configure the clock to use the the internal oscillator; therefore I have put one here.